tests/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
-include_directories(. googletest/include googletest)
-
-# enable_language(Fortran)
+include_directories(. googletest/include googletest ../utils ../host_reference)
 
 # enable tests build a common library for all test utilities
-set(QUDA_TEST_COMMON googletest/src/gtest-all.cc test_util.cpp test_params.cpp misc.cpp face_gauge.cpp)
+set(QUDA_TEST_COMMON googletest/src/gtest-all.cc)
```

Comment:
if all that is leftover here is googletest maybe we can clean that up. But we can clean that up later and just file an issue.